### PR TITLE
countdown: init at 1.0.0

### PR DESCRIPTION
**Pull request by oscarcarlsson**
###### Motivation for this change
New package for a simple, terminal based countdown program.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/countdown/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+buildGoPackage rec {
+  name = "countdown-${version}";
```
Comment (mmahut) [inline pkgs/applications/misc/countdown/default.nix]: Please consider using ``pname``.

File: pkgs/applications/misc/countdown/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    rev = "v${version}";
+    sha256 = "0pdaw1krr0bsl4amhwx03v2b02iznvwvqn7af5zp4fkzjaj14cdw";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/antonmedv/countdown;
+    description = "Terminal countdown timer";
+    maintainers = with maintainers; [ ];
```
Comment (mmahut) [inline pkgs/applications/misc/countdown/default.nix]: Are you sure you don't want to add yourself as a maintainer of this package?

**Comment by mmahut** [comment]
@GrahamcOfBorg build countdown

File: pkgs/applications/misc/countdown/default.nix (hunk 2 above)
Comment (oscarcarlsson) [reply pkgs/applications/misc/countdown/default.nix]: I've added myself as a maintainer for the package!

**Comment by mmahut** [comment]
Please squash your commits, thank you.

@GrahamcOfBorg build countdown

**Comment by oscarcarlsson** [comment]
> Please squash your commits, thank you.

I did, but no matter what I do I fail to make my maintainer-list.nix file and have it merge successfully. My latest attempt was to checkout the relevant file from master, add my changes and then squash everything together into one commit.

What would be a better way to fix this?

**Pull request closed without merging**